src/plugins/menu/menu.ts

```diff
@@ -298,70 +298,72 @@ export const renderMenu = (options: MenuOptions, editorView: EditorView, menuDom
   const updates: any[] = [];
 
   const toolbar = options.toolbar;
-
-  toolbar.forEach((group: ToolbarItem[], toolbarIndex: number): void => {
-    const isLastMenuGroup = toolbar.length - 1 === toolbarIndex;
-
-    group.forEach((toolbarItem: ToolbarItem, menuIndex: number): void => {
-      const isLastMenuItem = group.length - 1 === menuIndex;
-
-      // render dropdown
-      if (typeof toolbarItem === 'object') {
-        Object.keys(toolbarItem).forEach((dropdownGroup: ToolbarDropdownGroupKeys) => {
-          if (DROPDOWN_ITEMS.has(dropdownGroup)) {
-            const dropdown: ToolbarDropdownGroupValues = toolbarItem[dropdownGroup];
-
-            const dropdownView = new DropDownView(dropdownGroup, dropdown, editorView, options);
-            const rendered = dropdownView.render();
-            updates.push(rendered.updates);
-            menuDom.appendChild(rendered.dom);
-          } else {
-            console.warn('Unkown dropdown group:', dropdownGroup);
+  if (toolbar !== null) //handling excess div space rendered if toolbar is not to be rendered (at all).
```

Comment:
you could just return when it is null or undefined. so that changes will be minimal.

Reply:
Hi, I have made changes addressing this.